packages/card-react/src/NavCard.tsx

```diff
@@ -0,0 +1,67 @@
+import React, { FC, HTMLAttributes } from "react";
+import cn from "classnames";
+import { ErrorTag, InfoTag, SuccessTag, Tag, WarningTag } from "@fremtind/jkl-tag-react";
+import { Image } from "@fremtind/jkl-image-react";
+import { PaddingOptions } from "./types";
+
+// TODO: Eksporter typer fra Image-komponenten
+type ImageProps = React.ComponentProps<typeof Image>;
+
+export type TagType = "success" | "warning" | "info" | "error";
+export interface NavCardProps extends PaddingOptions, HTMLAttributes<HTMLAnchorElement> {
+    tag?: {
+        /**
+         * @default <none> Rendres som nøytral tag
+         */
+        type?: TagType;
+        text: string;
+    };
+    title: string;
+    href: string;
+    description?: string;
+    image?: Omit<ImageProps, "className">;
+    className?: string;
+}
+
+const getTag = (type?: TagType) => {
+    switch (type) {
+        case "info":
+            return InfoTag;
+        case "success":
+            return SuccessTag;
+        case "warning":
+            return WarningTag;
+        case "error":
+            return ErrorTag;
+        default:
+            return Tag;
+    }
+};
+
+export const NavCard: FC<NavCardProps> = React.forwardRef<HTMLAnchorElement, NavCardProps>(
+    ({ padding = "l", topPadding, image, tag, title, description, children, className, ...anchorProps }, ref) => {
+        const CardTag = getTag(tag?.type);
+        const spacingClass = `jkl-spacing-${padding}--all`;
+        const topSpacingClass = topPadding ? `jkl-spacing-${topPadding}--top` : "";
```

Comment:
Jeg er fortsatt ikke helt venn med disse. Er gjort som det er for å følge begrensningene satt i skissene, men det blir litt mange props. Alternativ er å kun ha en prop for padding, og implementere den som faktisk padding på innholdet ved hjelp av variant-klasser i stedet for å bruke hjelpeklassene for margin.

En bakdel med den tilnærmingen er at innholdet er i en wrapper, og man er dermed nødt til å sette stil på den hvis man vil gjøre noe custom med spacing/padding hos klient. Tar gjerne imot forslag her!

Reply:
Om du vil redusere antallet props på rotnivå kan det være et alternativ å endre `PaddingOptions.padding` til noe sånt som det her, inspirert av CSS shorthand for padding:

```ts
type BasePadding = "m" | "l" | "xl";

export interface PaddingShorthand {
  /**
   * Legger til ekstra rom i toppen av kortet, fra spacing-skalaen til Jøkul.
   * Ikke bruk ekstra topPadding sammen med tag. Verdier lavere enn for venstre og høyre padding
   * blir ignorert.
   * @default "l" (24px)
   */
  top?: SpacingStep;
  /** 
   * Bruk stegene "m", "l" eller "xl" fra spacing-skalaen til Jøkul
   * @default "l" (24px)
   */
  right?: BasePadding;
   /**
   * Legger til ekstra rom i bunnen av kortet, fra spacing-skalaen til Jøkul.
   * Verdier lavere enn for venstre og høyre padding blir ignorert.
   * @default "l" (24px)
   */
  bottom?: SpacingStep;
  /** 
   * Bruk stegene "m", "l" eller "xl" fra spacing-skalaen til Jøkul
   * @default "l" (24px)
   */
  left?: BasePadding;
}


export interface PaddingOptions {
   /**
    * Hvor mye rom skal det være rundt innholdet i kortet?
    * @default "l" (24px)
    */
   padding?: "m" | "l" | "xl" | PaddingShorthand;
 }
```

Reply:
Liker forslaget til William. Forresten, kanskje ikke så lurt å bruke jkl-spacing klassene her ettersom de teknisk sett er margins (og ikke padding)? Blir muligens vanskeligere å endre også hvis du plutselig må gjøre responsiv-design

Reply:
Grunnen til at jeg har brukt spacing-klassene er for å unngå tusen variant-klasser 😅 Ellers helt enig at det burde være padding når prop-en heter det. Men er også fan av forslaget fra @wkillerud her!

Reply:
Skjønner! 😄  Egentlig synes jeg det bør være opptil brukeren å definere padding, altså en default padding som kan overstyres med en classname. Men jeg skjønner at du vil sette begrensninger her sånn at det ikke blir for ulikt. Jeg ser for meg at vi får en mer sofistikert spacing-skala i fremtiden, der vi har spacing klasser/variabler som brukes spesifikt til dette formålet, altså padding i bokser (og ikke bare generiske spacing-verdier som i dagens løsning). Jeg vet det ble nevnt at det skulle gjøres noe med da vi refaktorerte spacing-skalaen, men kanskje dette er en fin tidspunkt for designere å komme med et forslag som vi kan utvide spacingskalaen med. Eller så kunne vi muligens laget en issue på dette, spacingskala med forskjellige type spacing brukt til forskjellige formål. Dette er sikkert ikke den første og siste komponenten som vil ha margin/padding-props.

Reply:
Kan bare se bort fra kommentaren min ettersom vi tok det opp i designforum